packages/x-components/src/x-modules/experience-controls/store/actions/fetch-controls.action.ts

```diff
@@ -1,37 +1,20 @@
 import { ExperienceControlsXStoreModule } from '../types';
+import { XPlugin } from '../../../../plugins/x-plugin';
 
 /**
  * Default implementation for the {@link ExperienceControlsActions.fetchControls}.
  *
  * @param _context - The {@link https://vuex.vuejs.org/guide/actions.html | context} of the actions,
  * provided by Vuex.
- * TODO: update when adapter is updateds.
- * @param request
- * @returns
+ *
+ * @param request - Parameters are empty for this action.
+ * @returns 'controls' data with the controls data configuration.
```

Comment:
You have to return both the controls and the events, basically the whole adapter response.